### ISSUE TITLE
Getting started notebook fails

### ISSUE THREAD

**Issue by krlng** [deepdoctection/deepdoctection#26]
**Bug** 💥
Getting started notebooks fails with:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [9], in <cell line: 2>()
      1 doc=iter(df)
----> 2 page = next(doc)

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/dataflow/dataflow/common.py:315, in MapData.__iter__(self)
    314 def __iter__(self):
--> 315     for dp in self.ds:
    316         ret = self.func(copy(dp))  # shallow copy the list
    317         if ret is not None:

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/dataflow/dataflow/common.py:315, in MapData.__iter__(self)
    314 def __iter__(self):
--> 315     for dp in self.ds:
    316         ret = self.func(copy(dp))  # shallow copy the list
    317         if ret is not None:

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/dataflow/dataflow/common.py:315, in MapData.__iter__(self)
    314 def __iter__(self):
--> 315     for dp in self.ds:
    316         ret = self.func(copy(dp))  # shallow copy the list
    317         if ret is not None:

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/dataflow/dataflow/common.py:316, in MapData.__iter__(self)
    314 def __iter__(self):
    315     for dp in self.ds:
--> 316         ret = self.func(copy(dp))  # shallow copy the list
    317         if ret is not None:
    318             yield ret

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/deepdoctection/pipe/base.py:89, in PipelineComponent.pass_datapoint(self, dp)
     87     with timed_operation(self.__class__.__name__):
     88         self.dp_manager.datapoint = dp
---> 89         self.serve(dp)
     90 else:
     91     self.dp_manager.datapoint = dp

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/deepdoctection/pipe/text.py:83, in TextExtractionService.serve(self, dp)
     81 assert predictor_input is not None
     82 width, height = None, None
---> 83 detect_result_list = self.predictor.predict(predictor_input)  # type: ignore
     84 if isinstance(self.predictor, PdfMiner):
     85     width, height = self.predictor.get_width_height(predictor_input)  # type: ignore

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/deepdoctection/extern/tessocr.py:274, in TesseractOcrDetector.predict(self, np_img)
    267 def predict(self, np_img: ImageType) -> List[DetectionResult]:
    268     """
    269     Transfer of a numpy array and call of pytesseract. Return of the detection results.
    270 
    271     :param np_img: image as numpy array
    272     :return: A list of DetectionResult
    273     """
--> 274     detection_results = predict_text(
    275         np_img,
    276         supported_languages=self.config.LANGUAGES,
    277         text_lines=self.config.LINES,
    278         config=config_to_cli_str(self.config, "LANGUAGES", "LINES"),
    279     )
    280     return detection_results

File ~/Library/Caches/pypoetry/virtualenvs/challenges-Ho27vFAF-py3.8/lib/python3.8/site-packages/deepdoctection/extern/tessocr.py:214, in predict_text(np_img, supported_languages, text_lines, config)
    202 all_results = []
    204 for caption in zip(
    205     results["left"],
    206     results["top"],
   (...)
    212     results["line_num"],
    213 ):
--> 214     if int(caption[4]) != -1:
    215         word = DetectionResult(
    216             box=[caption[0], caption[1], caption[0] + caption[2], caption[1] + caption[3]],
    217             score=caption[4] / 100,
   (...)
    222             class_name=names.C.WORD,
    223         )
    224         all_results.append(word)

ValueError: invalid literal for int() with base 10: '95.345306'
```

**Desktop (please complete the following information, if any other than the one in the install requirements):**
 - OS: 12.2.1

**Additional context** 🧬
Used `make install-dd-pt` to install from source as well as brew install tesseract-lang. Also installed the missing torch-vision dependency.

**Comment by JaMe76** [comment]
Thank you for reporting! 

I was able to reproduce the issue and have provided a fix with cf26a0c .

**Comment by JaMe76** [comment]
Closing with 0d840f7
